src/Minter/Library/Http.php

```diff
@@ -18,14 +18,16 @@
     protected $client;
 
     /**
-     * set base api url
+     * Set base API url.
      *
      * @param string $url
      */
-    protected function setApiUrl(string $url) : void {
-
+    protected function setApiUrl(string $url): void
+    {
         $this->client = new Client([
-            'base_uri' => $url
+            'base_uri' => $url,
+            'connect_timeout' => 15.0,
+            'timeout' => 30.0,
```

Comment:
Maybe this should be a little greater than 30 secs.
```suggestion
            'timeout' => 60.0,
```